otx/algorithms/detection/adapters/mmdet/utils/config_utils.py

```diff
@@ -241,6 +241,10 @@ def update_pipeline(cfg):
             continue
         config.data[f"{subset}_dataloader"] = config.data.get(f"{subset}_dataloader", ConfigDict())
 
+        remove_from_config(config.data[subset], "ann_file")
+        remove_from_config(config.data[subset], "img_prefix")
+        remove_from_config(config.data[subset], "classes")  # Get from DatasetEntity
```

Comment:
Why normal detection datasets works without these config modification?

Reply:
We can. But I don't prefer passing unnecessary kwargs to objet classes. I think the better way to solve this is to remove ann_file, img_prefix and classes from data pipeline (it's better to solve it from the root than patching).